build.fsx

```diff
@@ -1,3 +1,5 @@
+System.IO.Directory.SetCurrentDirectory __SOURCE_DIRECTORY__
+
```

Comment:
I think after opening the namespaces works as well and makes the code shorter :).